src/components/signUp/components/SignUp.jsx

```diff
@@ -2,11 +2,15 @@ import React from 'react';
 import { useSelector } from 'react-redux';
 import { SignUpForm } from './SignUpForm';
 import { MiniHeader, TextLink, Spinner } from '../../common/components';
+import { Helmet } from 'react-helmet';
 
 export const SignUp = () => {
     const { loading } = useSelector(state => state.signUp);
     return (
         <main className="bg-image h-screen w-auto bg-cover bg-no-repeat bg-top">
+            <Helmet>
+              <title>{ "Registration Page" }</title>
+            </Helmet>
```

Comment:
Same as above. Sign Up can be used instead of Registration Page. Move the Helmet Logic to the pages directory